src/components/app/Details.js

```diff
@@ -18,7 +17,6 @@ import { NetworkChart } from 'firefox-profiler/components/network-chart/';
 import { FlameGraph } from 'firefox-profiler/components/flame-graph/';
 import { JsTracer } from 'firefox-profiler/components/js-tracer/';
 import { selectSidebar } from 'firefox-profiler/components/sidebar';
-
```

Comment:
The changes after this file is not about this PR. It looks like these are reverting some files still. Can you please revert the changes after this line too?